templates/bird.conf.j2

```diff
@@ -29,6 +29,7 @@ protocol kernel {
   import {% if bird_kernel_ipv4_import_filter is defined %}filter {{ bird_kernel_ipv4_import_filter }}{% else %}all{% endif %};
   export {% if bird_kernel_ipv4_export_filter is defined %}filter {{ bird_kernel_ipv4_export_filter }}{% else %}none{% endif %};
   merge paths;
+  kernel table {% if bird_kernel_table is defined %} {{ bird_kernel_table }}{% else%}254{% endif %};
```

Comment:
This setting should not be present if the variable is undefined, so can you change this to:
{% if bird_kernel_table is defined %}kernel table {{ bird_kernel_table }}{% else %}254{% endif %};